README.md

```diff
@@ -765,6 +766,44 @@ expect(selectInput).not.toHaveValue(['second', 'third'])
 
 <hr />
 
+### `toHaveDisplayValue`
+
+```typescript
+toHaveDisplayValue(value: string | string[])
+```
+
+This allows you to check whether the given `<select>` element has the specified
+value.
+
+#### Examples
+
+```html
+<select data-testid="input-select">
+  <option value="">Select a fruit...</option>
+  <option value="banana">Banana</option>
+  <option value="ananas">Ananas</option>
+  <option value="avocado">Avocado</option>
+</select>
+<select data-testid="input-multiple">
+  <option value="">Select a fruit...</option>
+  <option value="banana" selected>Banana</option>
+  <option value="ananas">Ananas</option>
+  <option value="avocado" selected>Avocado</option>
+</select>
+```
+
+##### Using DOM Testing Library
+
+```javascript
+const inputSelect = getByTestId('input-select')
```

Comment:
We don't need the test-id here. Rather add an `aria-label` and use 
```suggestion
const inputSelect = getByRole('combobox', { name: 'what you wrote in aria-label' })
```

Reply:
Oh! I'm sorry! I copied the code from the tests T_T

I can use real labels if you prefer, like:

```html
<label for="single-select-example">Fruit</label>
<select id="single-select-example">
  <option value="">Select a fruit...</option>
  <option value="banana">Banana</option>
  <option value="ananas">Ananas</option>
  <option value="avocado">Avocado</option>
</select>

<label for="multiple-select-example">Fruits</label>
<select id="multiple-select-example">
  <option value="">Select a fruit...</option>
  <option value="banana" selected>Banana</option>
  <option value="ananas">Ananas</option>
  <option value="avocado" selected>Avocado</option>
</select>
```

```javascript
const inputSelect = getByLabelText('Fruit')
const inputMultiple = getByLabelText('Fruits')

expect(inputSelect).toHaveDisplayValue('Select a fruit...')
expect(inputMultiple).toHaveDisplayValue(['Banana', 'Avocado']
```

Reply:
No need to apologize. This API is new. ByTestId is an escape hatch and we want to avoid it wherever possible.